subsys/fs/CMakeLists.txt

```diff
@@ -4,18 +4,21 @@ if(CONFIG_FILE_SYSTEM)
   zephyr_interface_library_named(FS)
   #zephyr_link_interface_ifdef(CONFIG_FAT_FILESYSTEM_ELM ELMFAT)
   #zephyr_link_interface_ifdef(CONFIG_FILE_SYSTEM_NFFS   NFFS)
+  #zephyr_link_interface_ifdef(CONFIG_FILE_SYSTEM_LITTLEFS LITTLEFS)
```

Comment:
Any idea why all this `zephyr_link_interface_ifdef ` are commented out?

Reply:
Looks like @nashif disabled them when moving the previous file systems to a module.  I can remove them in a second commit.